pkg/band/as_923.go

```diff
@@ -18,11 +18,15 @@ import (
 	"go.thethings.network/lorawan-stack/pkg/ttnpb"
 )
 
+//revive:disable:var-naming
```

Comment:
Can we suppress this outside the code?

Reply:
Only per-file, not per-line/per-plugin.
I think in general we have very few places where we need to disable linting in this way(this package maybe actually the only one) and it may be possible to configure the plugin to actually ignore these https://github.com/mgechev/revive#var-naming.
I will take a closer look at it after this ground-work PR is merged.